lib/RTCPeerConnection.js

```diff
@@ -16,6 +16,7 @@ var merge = require('merge'),
 		REGEXP_NORMALIZED_CANDIDATE: new RegExp(/^candidate:/i),
 		REGEXP_FIX_CANDIDATE: new RegExp(/(^a=|\r|\n)/gi),
 		REGEXP_RELAY_CANDIDATE: new RegExp(/ relay /i),
+		REGEXP_HOST_CANDIDATE: / typ host/i,
```

Comment:
The literal regexp already returns a `RegExp` instance. So what you're doing is basically taking a `RegExp` and constructing a new instance of it - 2 instances in total.

I'll update the PR with a commit that removes all unnecessary `RegExp` constructions.